control/trajectory_follower/design/pid_longitudinal_controller-design.md

```diff
@@ -141,7 +92,59 @@ Depending on the actuating principle of the vehicle, the mechanism that physical
 
 In this controller, the predicted ego-velocity and the target velocity after the delay time are calculated and used for the feedback to address the time delay problem.
 
-### Parameter description
+### Slope compensation
```

Comment:
Where did you get these sentences for slope compensation? I thought it already exists in the longitudinal controller doc, but seems it was not.

Reply:
They are from [trajectory_follower-pid-design.md](https://github.com/autowarefoundation/autoware.universe/pull/1591/files#diff-92459c6c65d67958c719aa63a6aa6e4b5615235a8c42cd61d7fdbf5753f4e53c)
Since there were duplicated sentence, I merged them to one document about PID.

Reply:
thanks!